pom.xml

```diff
@@ -143,8 +143,11 @@
     <groovy-maven-plugin.version>2.0</groovy-maven-plugin.version>
     <surefire-plugin.version>2.20</surefire-plugin.version>
     <failsafe-plugin.version>2.20</failsafe-plugin.version>
+    <maven-compiler-plugin.version>3.6.1</maven-compiler-plugin.version>
```

Comment:
Aside from the coupled `google-clients.version` I would favor naming things after their `artifactId` to make things completely obvious and automatic.